custom/icds_reports/data_pull/exceptions.py

```diff
@@ -1,2 +1,6 @@
 class UnboundDataPullException(Exception):
     pass
+
+
+class DataPullInProgressError(Exception):
```

Comment:
just curious - do you have a mental model for when to use `Exception` versus `Error`?

Reply:
I would consider exception as something raised during code execution and error would be an issue with the process or non-code related issues.